bundles/org.openhab.binding.shelly/README.md

```diff
@@ -1178,6 +1178,33 @@ Refer to [Smartify Roller Shutters with openHAB and Shelly](doc/UseCaseSmartRoll
 |       | totalKWH     | Number   | yes       | Total energy consumption in kwh since the device powered up (resets on restart)   |
 |       | lastUpdate   | DateTime | yes       | Timestamp of the last measurement                                                 |
 
+### Shelly Plus Dimmer 10v(thing-type: shellyplus10v)
+
+| Group | Channel      | Type     | read-only | Description                                                                       |
```

Comment:
I would expect also a "device" group considering the thing type definition.

Reply:
the device channel group is defined under general notes, otherwise I need to add redundant information to each thing

Reply:
Ok